autoPyTorch/pipeline/components/training/trainer/StandardTrainer.py

```diff
@@ -56,6 +56,8 @@ def get_properties(dataset_properties: typing.Optional[typing.Dict[str, typing.A
     def get_hyperparameter_search_space(dataset_properties: typing.Optional[typing.Dict] = None,
                                         weighted_loss: typing.Tuple[typing.Tuple, bool] = ((True, False), True)
                                         ) -> ConfigurationSpace:
+        weighted_loss = CategoricalHyperparameter("weighted_loss", choices=weighted_loss[0],
```

Comment:
I dont think there is a need to have this line as we are already making the hyperparameter inside the if statement, moreover, on the branch [refactor_development_regularization_cocktails](https://github.com/automl/Auto-PyTorch/tree/refactor_development_regularization_cocktails) we are dramatically updating the code for trainers. So, I'll suggest you to ignore this module for now

Reply:
Maybe I should learn what happened from the merge. This is also not my change.